install-with-linux.md

```diff
@@ -19,7 +19,7 @@
 This guide is for installing Brunch to a USB (or other disk) using Linux. This guide should be compatible with Linux Live USBs for quick installations as well. To begin, boot into the Linux distro of your choice and click the dropdown below to continue.
 
 <details>
-  <summary>Click to open Brunch USB guide</summary>
+  <summary>Click to open Brunch USB guide(Debian/Ubuntu)</summary>
```

Comment:
There are instructions for Arch Linux too, so the [Debian/Ubuntu] part is incorrect.